core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.ts

```diff
@@ -380,10 +380,6 @@ export class TopNavigationBarComponent implements OnInit, OnDestroy {
     return this.urlInterpolationService.getStaticImageUrl(imagePath);
   }
 
-  getOppiaBlogUrl(): string {
```

Comment:
Lets keep this as we don't insert links in HTML files directly for maintainability reasons.